src/sentry/static/sentry/app/types/index.tsx

```diff
@@ -868,7 +868,9 @@ type GroupActivityBase = {
 
 type GroupActivityNote = GroupActivityBase & {
   type: GroupActivityType.NOTE;
-  data: Record<string, any>;
+  data: {
+    text: string;
```

Comment:
Is there only ever one attribute?

Reply:
this is the only one that I could identify

Reply:
yes..we are using it only here https://github.com/getsentry/sentry/blob/b16bcd6da422eefe74100da25e52110cec583a2b/src/sentry/static/sentry/app/views/organizationGroupDetails/groupActivity.tsx#L181